sphinx_source/Ucuncu_taraf_moduller/django_modülü/django_2

```diff
@@ -0,0 +1,484 @@
+.. meta::
+   :description: Bu bölümde django API'ını öğreneceğiz.
+   :keywords: python, django, çeviri
+   
+.. highlight:: python3
+
+*********************************
+İLK DJANGO PROJENİ YAZ, part 2
+*********************************
+
+Veritabanı Kurulumu
+********************
+
+Şimdi **mysite/settings.py** dosyasını açın. Önce TIME_ZONE 
+değerini `Europe/Istanbul`  yaparak İstanbul'a ayarlayın.
+
+..Note:: Dosyanın en başındaki INSTALLED_APPS ayarına dikkat edin.
+         Projenizde kullandığınız appleri tutar. Yeni bir app 
+         kullanacağınız zaman bu listeye eklemelisiniz. 
+
+Varsayılan olarak gelen appler şunlardır::
+
+    django.contrib.admin -->Admin panelini oluşturur.
+    django.contrib.auth --> Bir kimlik doğrulama sistemi.
+    django.contrib.contenttypes -->İçerik türleri için bir framework
+    django.contrib.sessions --> Bir oturum frameworku
+    django.contrib.messages --> Bir mesajlaşma frameworku
+    django.contrib.staticfiles-->Statik dosyaları yönetmek için bir framework.
+
+Bu applerden bazıları en az bir veritabanı kullanıyor. Ancak bu veritabanlarının kullanılabilmesi için bir 
```

Comment:
Satır uzunlukları en fazla 80 karakter olmalı. Bu, metnin geri kalanı için de geçerlidir.